ld_script.txt

```diff
@@ -174,6 +174,7 @@ SECTIONS {
         src/record_mixing.o(.text);
         asm/record_mixing.o(.text);
         asm/sound_check_menu.o(.text);
+		src/secret_base_80bb4ac.o(.text);
         asm/secret_base.o(.text);
         src/secret_base.o(.text);
```

Comment:
make this `src/secret_base.o(.text_b);`, and then add this line before the function already in there: `asm(".section .text_d");`. That way you can just put your stuff in secret_base.c